### PR TITLE
[Docs]: Linked to Organisation Contribution Guide

### LINKED ISSUE #160: Remove CONTRIBUTING.rst as it's superseded by .github
Remove contributing guidance from this repo because it is now contained in the organisation level [.github repo](https://github.com/Transport-for-the-North/.github.git)

### PR DESCRIPTION
**Pull request by MattBuckley-TfN**
Describe Changes
---

Removed local CONTRIBUTING.rst because it is superseded by the organisation level one, which is linked in README.
Removed local PR template as this is superseded by the organisation level one.

- Closes #160 

Task Checklist
----
- [x] Have unittests been added (testing individual functions and their edge cases) - NA
- [x] Have integration tests been added (if applicable, to test modules of functionality) - NA
- [x] Have new dependencies been added to either `requirements.txt`, `requirements_dev.txt`, or `docs/requirements.txt`? - NA

----
📚 Documentation preview 📚: https://caftoolkit--161.org.readthedocs.build/en/161/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,8 +21,19 @@ a lot of the useful stuff from
...
+This package is sits within the [Common Analytical Framework (CAF)](https://transport-for-the-north.github.io/caf_homepage/intro.html),
+which is a collaboration between transport bodies in the UK to develop and maintain commonly used
+transport analytics and appraisal tools.
 
 ## Contributing
 CAF.Toolkit happily accepts contributions.
 
-The best way to contribute to this project is to go to the issues tab and report bugs or submit a feature request. This helps CAF.Toolkit become more stable and full-featured. Please check the closed bugs before submitting a bug report to see if your question has already been answered.
+The best way to contribute to this project is to go to the [issues tab](https://github.com/Transport-for-the-North/caf.toolkit/issues)
+and report bugs or submit a feature request. This helps CAF.toolkit become more
```
Comment (BenTaylor-TfN) [inline README.md:34]:
```suggestion
to report bugs or submit feature requests. This helps CAF.toolkit become more
```

**Review by BenTaylor-TfN — APPROVED** [review]
